data/re.sonny.Commit.appdata.xml

```diff
@@ -2,7 +2,7 @@
 <component type="desktop">
   <id>re.sonny.Commit</id>
   <launchable type="desktop-id">re.sonny.Commit.desktop</launchable>
-  <name>Commit</name>
+  <name translatable="no">Commit</name>
```

Comment:
Does it make sense to mark the app name as non translatable? 

It's still translatable in the `.desktop` file - not sure if there is  way to prevent it as well.

Reply:
Sometimes it makes sense to leave the application name translatable. For example, [Drawing](https://github.com/maoschanz/drawing) (Kreslení) or [Dialect](https://github.com/dialect-app/dialect) (Dialekt). In the case of the name "Commit", where it's actually a git keyword, it probably doesn't need to be translatable. If you don't want the name to be translated, maybe you can probably just add the comment "TRANSLATORS: Don't translate".

Reply:
Makes sense, I tried to found out if git uses a different keyword in other languages but it doesn't seem to be the case.